packages/reporter/src/main.tsx

```diff
@@ -15,10 +17,10 @@ import scroller, { Scroller } from './lib/scroller'
 import statsStore, { StatsStore } from './header/stats-store'
 import shortcuts from './lib/shortcuts'
 
-import Header from './header/header'
+import Header, { ReporterHeaderProps } from './header/header'
 import Runnables from './runnables/runnables'
 
-export interface ReporterProps {
+export type ReporterProps = {
```

Comment:
Should we just make two interfaces to make it more clear?

`BaseReporterProps` could hold the common ones, then have `MultiSpecReporterProps` and `SingleSepcReporterProps`?

Reply:
Sounds good yeah. I just copied the code as it was but it feels better with interfaces

Reply:
better?

Reply:
I was thinking this too but didn’t feel confident enough in my Ts to suggest it